ReqIFSharp/AccessControlledElement.cs

```diff
@@ -52,9 +47,13 @@ public override void ReadXml(XmlReader reader)
             base.ReadXml(reader);
             var isEditable = reader.GetAttribute("IS-EDITABLE");
 
-            if (isEditable != null)
+            if (isEditable == "1" || isEditable == "true")
```

Comment:
> Also please make use of the XmlConvert class  

Bad idea, because it's throw exceptions. You can look on [referencesource](https://referencesource.microsoft.com/#System.Xml/System/Xml/XmlConvert.cs,755ad86ddf2db909) how work XmlConverter

My code also guarantees locale-independent. Or we can use try {} catch() in every place in program, but I don't think that is good.